InMemoryFeedStore/InMemoryFeedStore.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  InMemoryFeedStore.swift
+//  FeedStoreChallenge
+//
+//  Created by Hugo Alonso on 05/02/2021.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import Foundation
+
+public class InMemoryFeedStore: FeedStore {
+	private var cache: CacheData?
+
+	public static let shared: InMemoryFeedStore = InMemoryFeedStore()
```

Comment:
The compiler can infer the type:

```suggestion
	public static let shared = InMemoryFeedStore()
```